db/seeds.rb

```diff
@@ -5,3 +5,36 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+# [].tap do |array|
+#   100.times do |i|
+#     created_at = Time.current
+
+#     array << {
+#       title: "This is #{i+1} Article",
+#       text: "Hello This is #{i+1} Text",
+#       created_at: created_at,
+#       updated_at: created_at
+#     }
+#   end
+
+#   result = Article.insert_all array
+#   puts result.inspect
+
+100.times do |i|
+  created_at = Time.current
+  result = Article.insert_all(
+    [
+      {
+        title: "This is #{i+1} Article",
+        text: "This is #{i+1} Text",
```

Comment:
create meaningful name of article and description

Reply:
Use this fake data https://deeporion.slack.com/archives/C03QRV1H1GR/p1662395344111339